R/utils.R

```diff
@@ -1,5 +1,8 @@
 format_triad <- function(triad) {
-  check_vector(triad, "", length = 3, unique = TRUE)
+  chk_vector(triad)
+  check_values(triad, "")
+  check_dim(triad, dim = length(3))
```

Comment:
This should be `check_dim(triad, dim = 3)`. The current code is equivalent to `check_dim(triad, dim = 1)` as `length(3)` is `1`